src/client/src/index.js

```diff
@@ -8,14 +8,14 @@ import './setup/css';
 // }
 
 (async () => {
-  const slidesInfo = fetchSlides(require.context(process.env.SLIDE_PATH));
+  const slidesInfo = fetchSlides(require.context(process.env.SLIDE_PATH, false, /\.md$/));
```

Comment:
I think files that file-loader emits causes #86 and we need only "*.md" files to crate `slidesInfo`. How about this?

Reply:
Good catch! You're right.